sdk/endpoints/online/managed/debug-online-endpoints-locally-in-visual-studio-code.ipynb

```diff
@@ -0,0 +1,166 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Debug online endpoints locally in Visual Studio Code"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Launch development container"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.ai.ml.entities import ManagedOnlineEndpoint, ManagedOnlineDeployment\n",
+    "from azure.identity import DefaultAzureCredential, AzureCliCredential"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "subscription_id = \"<YOUR-SUBSCRIPTION-ID>\"\n",
+    "resource_group = \"<YOUR-RESOURCE-GROUP>\"\n",
+    "workspace_name = \"<YOUR-WORKSPACE>\"\n",
+    "\n",
+    "endpoint_name = \"<YOUR-ENDPOINT-NAME>\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "credential = AzureCliCredential()\n",
+    "ml_client = MLClient(\n",
+    "    credential,\n",
+    "    subscription_id=subscription_id,\n",
+    "    resource_group_name=resource_group,\n",
+    "    workspace_name=workspace_name\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = ManagedOnlineEndpoint(name=endpoint_name)\n",
+    "\n",
+    "endpoint = ml_client.online_endpoints.begin_create_or_update(endpoint)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_path = \"endpoints/online/managed/sample/blue-deployment.yml\"\n",
+    "deployment = ManagedOnlineDeployment.load(file_path)\n",
```

Comment:
I believe users who want to debug online endpoints locally in VSCode with Python also want to define deployment by Python instead of the YAML file. 
Can we replace the two lines with codes like following? (Just an example)

```python
model = Model(path="../model-1/model/sklearn_regression_model.pkl")
env = Environment(
    conda_file="../model-1/environment/conda.yml",
    image="mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1",
)

deployment = ManagedOnlineDeployment(
    name="blue",
    endpoint_name=endpoint_name,
    model=model,
    environment=env,
    code_configuration=CodeConfiguration(
        code="../model-1/onlinescoring", scoring_script="score.py"
    ),
    instance_type="Standard_DS2_v2",
    instance_count=1,
)
```